modules/videoio/test/test_ffmpeg.cpp

```diff
@@ -536,5 +536,17 @@ TEST(videoio_ffmpeg, create_with_property_badarg)
     EXPECT_FALSE(cap.isOpened());
 }
 
+// related issue: https://github.com/opencv/opencv/issues/16821
+TEST(videoio_ffmpeg, open_from_web)
```

Comment:
Lets disable this test before merge (as it relies on Internet resource)
It is OK to run it locally using `--gtest_also_run_disabled_tests`

```
TEST(videoio_ffmpeg, DISABLED_open_from_web)
```